backend/app/core/exceptions_handler.py

```
@@ -0,0 +1,49 @@
+    exc: RequestValidationError
+):
+    return ResponseHandler.error_response(
+        message="Validation Error",
+        code=422,
+        errors={
+            "details": exc.errors(),
+            "path": request.url.path,
+        },
+    )
+
+async def internal_exception_handler(request: Request, exc: Exception):
```

Ruff (0.14.10)

Unused function argument: `exc`

(ARG001)